cmd/agent/subcommands/run/command.go

```diff
@@ -336,6 +337,7 @@ func getSharedFxOption() fx.Option {
 			path.DefaultStreamlogsLogFile,
 		)),
 		flare.Module(),
+		lsof.Module(),
```

Comment:
I think we could probably move this directly in the flare component. I think is a bit overkill to have a component that only add a flare provider. 

What do you think?

Reply:
I was thinking long term, this component could provide an API endpoint to access the open files information, that way the flare can include the info for every running agent

Reply:
Good use case 😄

Reply:
If that is the case we might need to make part of the core bundle, what do you think?